work/lessonbuilder_highlight_external_links.py

```diff
@@ -45,7 +45,7 @@ def run(SITE_ID, APP):
 
             for link in APP['external_links']:
```

Comment:
Please rename the config key external_links ("external" isn't clear - external to what? and it's not matching links exactly either). It seems to be a list of domains to match so maybe lessons_highlight_domains and likewise rename vula_tools to lessons_highlight_names